api-admin/src/main/resources/api/openapi.yml

```diff
@@ -1173,6 +1173,41 @@ paths:
                 type: object
         '500':
           $ref: '#/components/responses/ServerError'
+  /api/internal/v1/populate/portal/upload:
```

Comment:
The addition of this second populate endpoint makes the original `POST /api/internal/v1/populate/portal` endpoint a little more mysterious. Could you also update the summary in the swagger spec for that endpoint to make its purpose more clear?